screengrab/lib/screengrab/runner.rb

```diff
@@ -146,7 +146,7 @@ def determine_internal_screenshots_paths(app_package_name, locales)
         "/data/user/0/#{app_package_name}/files/#{app_package_name}/screengrab/#{locale}/images/screenshots"
       end
```

Comment:
```suggestion
        [
          "/data/user/0/#{app_package_name}/files/#{app_package_name}/screengrab/#{locale}/images/screenshots",
          
          # https://github.com/fastlane/fastlane/issues/15653#issuecomment-578541663
          "/data/data/#{app_package_name}/files/#{app_package_name}/screengrab/#{locale}/images/screenshots"
        ]
      end.flatten
```

Reply:
Thank you for you time taking to review my PR!

I have checked suggested changes locally and it seems to me this creates duplicates in resolved path. Since `determine_internal_screenshots_paths` unconditionally is called with array of locales. And test run for second locale resolves both first and second locale paths as valid. Example output from emulator 29;google_apis;x86_64:

```shell
21:15:30]: Running tests for locale: de-DE
[21:15:30]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 shell am instrument --no-window-animation -w \
-e testLocale de_DE \
-e endingLocale en_US \
-e appendTimestamp true \
-e package screenshot.mocked \
de.flixbus.app.debug.test/de.meinfernbus.internal.FlixAndroidTestRunner
[21:15:59]: ▸ screenshot.mocked.RidesScreenShooter:.
[21:16:03]: ▸ screenshot.mocked.SearchScreenShooter:.
[21:16:07]: ▸ screenshot.mocked.TicketScreenShooter:.
[21:16:07]: ▸ Time: 31.46
[21:16:07]: ▸ OK (3 tests)
[21:16:08]: Pulling captured screenshots for locale de-DE from the device
ls: /sdcard/de.flixbus.app.debug/screengrab: No such file or directory
[21:16:08]: Exit status: 1
ls: /data/data/de.flixbus.app.debug/app_screengrab: No such file or directory
[21:16:08]: Exit status: 1
ls: /data/data/de.flixbus.app.debug/screengrab: No such file or directory
[21:16:08]: Exit status: 1
[21:16:08]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 pull /data/user/0/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/de-DE/images/screenshots /var/folders/ww/cgv63frx1c5fdzpn183d67wc0000gn/T/d20200303-4970-o2knv3
[21:16:08]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 pull /data/data/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/de-DE/images/screenshots /var/folders/ww/cgv63frx1c5fdzpn183d67wc0000gn/T/d20200303-4970-o2knv3
ls: /data/user/0/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/cs-CZ/images/screenshots: No such file or directory
[21:16:08]: Exit status: 1
ls: /data/data/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/cs-CZ/images/screenshots: No such file or directory
[21:16:08]: Exit status: 1
[21:16:08]: Screenshots copied to screenshots/de-DE/images/phoneScreenshots
[21:16:08]: Running tests for locale: cs-CZ
[21:16:08]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 shell am instrument --no-window-animation -w \
-e testLocale cs_CZ \
-e endingLocale en_US \
-e appendTimestamp true \
-e package screenshot.mocked \
de.flixbus.app.debug.test/de.meinfernbus.internal.FlixAndroidTestRunner
[21:16:33]: ▸ screenshot.mocked.RidesScreenShooter:.
[21:16:36]: ▸ screenshot.mocked.SearchScreenShooter:.
[21:16:39]: ▸ screenshot.mocked.TicketScreenShooter:.
[21:16:39]: ▸ Time: 16.567
[21:16:39]: ▸ OK (3 tests)
[21:16:39]: Pulling captured screenshots for locale cs-CZ from the device
ls: /sdcard/de.flixbus.app.debug/screengrab: No such file or directory
[21:16:39]: Exit status: 1
ls: /data/data/de.flixbus.app.debug/app_screengrab: No such file or directory
[21:16:40]: Exit status: 1
ls: /data/data/de.flixbus.app.debug/screengrab: No such file or directory
[21:16:40]: Exit status: 1
[21:16:40]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 pull /data/user/0/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/de-DE/images/screenshots /var/folders/ww/cgv63frx1c5fdzpn183d67wc0000gn/T/d20200303-4970-7tlqen
[21:16:40]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 pull /data/data/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/de-DE/images/screenshots /var/folders/ww/cgv63frx1c5fdzpn183d67wc0000gn/T/d20200303-4970-7tlqen
[21:16:40]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 pull /data/user/0/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/cs-CZ/images/screenshots /var/folders/ww/cgv63frx1c5fdzpn183d67wc0000gn/T/d20200303-4970-7tlqen
[21:16:40]: $ /usr/local/share/android-sdk/platform-tools/adb -s emulator-5554 pull /data/data/de.flixbus.app.debug/files/de.flixbus.app.debug/screengrab/cs-CZ/images/screenshots /var/folders/ww/cgv63frx1c5fdzpn183d67wc0000gn/T/d20200303-4970-7tlqen
[21:16:40]: Screenshots copied to screenshots/cs-CZ/images/phoneScreenshots
```
(check last 5 lines. fastlane pulls both de-DE and cs-CZ into screenshots/cs-CZ/images folder)

Reply:
Ahhh, this is another issue that we are trying to solve! So this is a good thing you saw this in a way 😊 I'm going to see if I can replicate this on my side 🤔